doc/modules/index.md

```diff
@@ -61,22 +64,29 @@ modules {
 If a path is a directory then Rspamd scans it for `*.lua` pattern and load all
 files matched.
 
-The following Lua modules are enabled in the default configuration (but may require additional configuration to work, see notes below):
+The following Lua modules are enabled in the default configuration. Except where noted with a trailing asterisk,
+each of these modules has its own module_name.conf file in $LOCAL_CONFDIR/modules.d. Some modules require
+additional configuration to work, see notes below. :
 
 - [antivirus](antivirus.html) - integrates virus scanners (requires configuration)
 - [arc](arc.html) - checks and signs ARC signatures
 - [asn](asn.html) - looks up ASN-related information
-- [clickhouse](clickhouse.html) - pushes scan-related information to clickhouse DBMS (requires configuration)
-- [bayes_expiry](bayes_expiry.html) - provides expiration of statistical tokens (requires Redis and configuration)
+- [clickhouse](clickhouse.html) - pushes scan-related information to the clickhouse DBMS (requires configuration)
+- [bayes_expiry](bayes_expiry.html) - provides expiration of statistical tokens (requires Redis and configuration) *
```

Comment:
The last `*` character looks like a misprint, isn't it?

Reply:
See line 67-69 about the asterisk. It looks like there is no bayes_expiry.conf file, and that might be the only Lua module that doesn't.

Would it be better to leave out "with a trailing asterisk" and then to be explicit that bayes_expiry requires the creation of a new bayes_expiry.config file?

Maybe that's an oversight in file system and you want to add an empty .config file?

Since we're in this area, we say "requires configuration" but I don't think there is any detail about what kind of configuration is required - it's left to the user to figure out each file and parameter. If that info is documented somewhere, I'll add a reference in here.

Reply:
Response posted in the code [here](https://github.com/rspamd/rspamd.com/pull/506/files/946a197c3364a5c41f911ee1eace0a508a1bcb1d#r638352480) (doesn't seem to be reflected in this pull thread).

Reply:
It doesn't require creation of a config file. `Bayes expiry` module configuration is a part of the classifier configuration. It is documented [here](https://github.com/rspamd/rspamd.com/blame/master/doc/modules/bayes_expiry.md#L12).